mods/creative/init.lua

```diff
@@ -1,14 +1,17 @@
 creative = {}
 
+minetest.register_privilege("creative", {"Allow player to use creative inventory",
```

Comment:
Yes, it's running on Xanadu for now where admin have 'creative' privelages and access to the creative inventory/placement features.